### PR TITLE
Add verification step for authorization docs

### PR DESCRIPTION
**Pull request by sheajamba**
## Description

Adds a verification step to the Trino security overview page with a link to the steps to verify system-level rules file.

## Release notes

( ) This is not user-visible or docs only and no release notes are required.
( ) Release notes are required, please propose a release note for me.
( ) Release notes are required, with the following suggested text:

```markdown
# Section
* Fix some things. ({issue}`issuenumber`)
```

### CODE REVIEW

File: docs/src/main/sphinx/security/file-system-access-control.rst [hunk 1]
```diff
@@ -331,6 +331,34 @@ The example below defines the following table access policy:
       ]
     }
 
+.. _verify_rules:
+
+Verify configuration
+^^^^^^^^^^^^^^^^^^^^
+
+To verify the system-access control file is configured properly, you should set
```
Comment (mosabua) [inline docs/src/main/sphinx/security/file-system-access-control.rst:339]: you set the rules to the following configuration:

File: docs/src/main/sphinx/security/file-system-access-control.rst [hunk 2]
```diff
@@ -331,6 +331,34 @@ The example below defines the following table access policy:
...
+    {
+      "catalogs": [
+        {
+          "catalog": "system",
+          "allow": "none"
+        }
+      ]
+    }
+
+From the Trino :doc:`CLI </client/cli>`, run query:
```
Comment (mosabua) [inline docs/src/main/sphinx/security/file-system-access-control.rst:353]: Restart your cluster to activate the cluster, and then from ...

File: docs/src/main/sphinx/security/file-system-access-control.rst [hunk 3]
```diff
@@ -331,6 +331,34 @@ The example below defines the following table access policy:
...
+
+.. code-block:: text
+
+  trino> SELECT * FROM system.runtime.nodes;
+  Query 20200824_183358_00000_c62aw failed: Access Denied: Cannot access catalog system
+
+Remove these rules and restart the Trino server.
```
Comment (mosabua) [inline docs/src/main/sphinx/security/file-system-access-control.rst:360]:
```suggestion
Remove these rules and restart the Trino cluster.
```

**Review by mosabua — CHANGES_REQUESTED** [review]

**Review by mosabua — APPROVED** [review]
Looks good now.

**Comment by mosabua** [comment]
@dain can you give this a quick glance and merge?

**Review by electrum — APPROVED** [review]

**Pull request merged**